src/index.html

```diff
@@ -5,9 +5,34 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="card">
+      <img
```

Comment:
apply recommendations from a [checklist](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)
<img width="239" alt="Screenshot 2023-11-14 at 22 10 18" src="https://github.com/mate-academy/layout_product-cards/assets/117587463/bbcc8526-e92a-4e96-9754-bb3125ffece4">

Reply:
not fixed https://github.com/mate-academy/layout_product-cards/pull/3764/files#r1393211302

Reply:
Not fixed

Reply:
If you have any problems with that pleas it in an fe_chat